js/testing_1_2_3.js

```diff
@@ -0,0 +1,14 @@
+// https://www.codewars.com/kata/testing-1-2-3/train/javascript
+
+var number=function(array){
+  let i = 1;
+  const newArray = array.map(function(el){
+     el = `${i}: ${el}`;
```

Comment:
Consider avoiding reassigning function parameters :smile: it's generally bad practice to reassign variables. That's the reason `const` is usually preferred over `let`/`var`. The same goes for parameters